src/main/java/wolox/training/models/Book.java

```diff
@@ -40,11 +42,13 @@ public class Book {
     @Column(nullable = false)
     private String isbn;
 
+    @ManyToMany(mappedBy = "books")
```

Comment:
You're gonna hate me for this 🥺  but the book should not have a list of users, since it's not an attribute of the book (the book doesn't care which users "have it" 😅 ) 
try removing this and let me know if something fails!